src/js/components/modals/CliInstallModal.js

```diff
@@ -77,11 +77,10 @@ class CliInstallModal extends React.Component {
 
     const instructions = [
       `curl ${downloadUrl} -o dcos`,
-      "sudo mv dcos /usr/local/bin",
-      "sudo chmod +x /usr/local/bin/dcos",
-      `dcos config set core.dcos_url ${clusterUrl}`,
-      Hooks.applyFilter("dcosInstallCommandExtraSteps", undefined),
-      "dcos"
+      `sudo mv dcos /usr/local/bin`,
+      `sudo chmod +x /usr/local/bin/dcos`,
+      `dcos cluster setup ${clusterUrl}`,
+      `dco`
```

Comment:
@weblancaster I'm not sure that is the case. We shouldn't assume a change in implementation and should always question cases like this. If you search for where `dcosInstallCommandExtraSteps` is defined, please note the additional commands that are added and please verify with @tamarrow. Thanks.